picasso/src/test/java/com/squareup/picasso/RequestCreatorTest.java

```diff
@@ -790,6 +797,28 @@ public void nullKeyTransformationInvalid() {
     });
   }
 
+  @Test(expected = IllegalArgumentException.class)
+  public void nullKeyInTransformationListInvalid() {
+    List<Transformation> transformations = new ArrayList<Transformation>();
+    transformations.add(new Transformation() {
+      @Override public Bitmap transform(Bitmap source) {
+        return source;
+      }
+
+      @Override public String key() {
+        return null;
+      }
+    });
+    new RequestCreator().transform(transformations);
+  }
+
+  @Test
+  public void transformationListImplementationValid() {
+    List<TestTransformation> transformations = new ArrayList<TestTransformation>();
+    transformations.add(new TestTransformation("test"));
+    new RequestCreator().transform(transformations);
```

Comment:
This test doesn't verify anything. We need to at least assert that each transformation in the list was included in the final request in the same order.